app/addons/config/routes.js

```diff
@@ -67,7 +67,7 @@ function (app, FauxtonAPI, Config, Views, CORSComponents, CORSActions, ClusterAc
       this.sidebar = this.setView('#sidebar-content', new Views.Tabs({
         sidebarItems: [
           {
-            title: 'Main config',
+            title: 'config.js File',
```

Comment:
Speaking from backend side, there are no Main config or 'config.js'. Users knowns that all configs are stored in INI files, so if you tell them about some config.js someone mind gonna blow.

"Main config" name isn't good here as well since as you can see, there is CORS menu item. Which means, that here we actually see some configuration groups, not some special configuration  From this point "General" instead of "Main config" would be more sane replacement. Or "All configuration" - also good, especially in mid-term perspective (imagine that there will be more groups like Auth, Compaction, Databases, Admins etc).